sdk/vc-data/src/data/courtRecordSearch/v1.ts

```diff
@@ -0,0 +1,92 @@
+import { VCV1, VCV1Subject } from '@affinidi/vc-common'
+
+import { PersonEV1, getBaseV1ContextEntries } from '../base'
+import {
+  CreateThing,
+  ExpandThing,
+  ExtendThing,
+  Type,
+  createContextEntry,
+  createVCContextEntry,
+  CreateExpandedThing,
+} from '../util'
+
+// Helper Types
+
+export type CourtRecordSearchQueryV1 = CreateThing<
+  'CourtRecordSearchQuery',
+  {
+    parent: PersonEV1
+    spouse?: PersonEV1
+    address: CreateExpandedThing<'PostalAddress'>
```

Comment:
I do still like the idea of a `PostalAddressRole` to contain the meta data about an address:

```ts
type PostalAddress = CreateExpandedThing<'PostalAddress'>
type PostalAddressRoleMixin = CreateThing<'PostalAddressRole', {
  address: PostalAddress
}>
type PostalAddressRole = ExtendThing<PostalAddressRoleMixin, CreateExpandedThing<'Role'>>

export type CourtRecordSearchQueryV1 = CreateThing<
  'CourtRecordSearchQuery',
  {
    parent: PersonEV1
    spouse?: PersonEV1
    address: MaybeArray<PostalAddressRole | PostalAddress>
  }
>
```

Reply:
yup we discussed yesterday but I'll add comment here too. addressStatus is being used to describe the query rather than the address